lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -275,9 +283,126 @@ bool PatchEntryPointMutate::lowerCpsOps(Function *func) {
     numUserdata = m_cpsShaderInputCache.getTypes().size();
   }
 
+  SmallVector<CpsExitInfo> exitInfos;
+  const DataLayout &layout = func->getParent()->getDataLayout();
```

Comment:
Can move that line a bit to the bottom, since `layout` is first used ~50 lines later if I'm not mistaken